src/integrations/Braze/nativeSdkLoader.js

```diff
@@ -0,0 +1,39 @@
+import { LOAD_ORIGIN } from '../../utils/ScriptLoader';
+import { BrazeOperationString } from './constants';
+
+// START-NO-SONAR-SCAN
+/* eslint-disable */
```

Comment:
These comments are not required anymore.
You can make the following changes to skip these static checks for this entire file:

- Add the line `src/integrations/**/nativeSdkLoader.js` in `.eslintignore`.
- Append the string `,src/integrations/**/nativeSdkLoader.js` to `sonar.exclusions`, `sonar.coverage.exclusions`, and `sonar.cpd.exclusions` lines in `sonar-project.properties` file.

Reply:
Addressed